xml/Microsoft.Extensions.Logging.Console/ConsoleFormatterOptions.xml

```diff
@@ -93,7 +93,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets indication whether or not UTC timezone should be used to for timestamps in logging messages. Defaults to <c>false</c>.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if the UTC time zone is used for timestamps in logging messages; otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
```